features/kinderen.feature

```diff
@@ -100,11 +100,23 @@ Functionaliteit: Kinderen van een persoon raadplegen
 
     Scenario: ontkenning ouderschap
       Gegeven de persoon met burgerservicenummer '000000061' heeft een 'kind' met de volgende gegevens
-      | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
-      | Daan              | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
+	  | naam                            | waarde          |
+      | voornamen (02.10)               | Daan            |
+      | voorvoegsel (02.30)             | de              |
+	  | geslachtsnaam (02.40)           | Vries           |
+	  | geboortedatum (03.10)           | 20031107        | 
+	  | gemeente document (82.10)       | 0518            | 
+	  | datum document (82.20)          | 20031109        | 
+	  | beschrijving document (82.30)   | PL gerelateerde | 
+	  | ingangsdatum geldigheid (85.10) | 20031107        | 
+	  | datum van opneming (86.10)      | 20031109        |
       En het 'kind' is gewijzigd naar de volgende gegevens
-      | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
-      | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+	  | naam                            | waarde          |
+      | gemeente document (82.10)       | 1926             |
+      | datum document (82.20)          | 20040105         | 
+	  | beschrijving document (82.30)   | D27894-2004-A782 | 
+	  | ingangsdatum geldigheid (85.10) | 20031107         | 
+	  | datum van opneming (86.10)      | 20040112         |
```

Comment:
Zie ook dba-dev scenario: Wat maakt dit een ontkenning ?

Vanuit functioneel perspectief: moet hier dan ook niet beschreven worden hoe omgegaan wordt met herroeping van adoptie, nietigverklaring van erkenning. En als dat hetzelfde is, nemen we dat dan op in de scenario-omschrijving ? 




Hier is de uitlijning niet goed.

Reply:
Voorstel aanpassen titel 'ontkenning ouderschap' naar 'ontkenning ouderschap, terugdraaien adoptie, ....'

@Johan komt met voorstel voor een gewijzigde titel.

"En het 'kind' is gewijzigd naar de volgende gegevens" moet worden veranderd naar "En het 'kind' is gecorrigeerd naar de volgende gegevens"

Reply:
In En stap is 'gewijzigd' aangepast naar 'gecorrigeerd'.

Reply:
Titel van het scenario 'ontkenning ouderschap' is gewijzigd in 'ontkenning ouderschap, herroeping van adoptie en nietig verklaren erkenning'.